examples/spring_boot/WORKSPACE

```diff
@@ -7,6 +7,8 @@ load("@rules_jvm_external//:defs.bzl", "maven_install")
 
 maven_install(
     artifacts = [
+        "org.apache.logging.log4j:log4j-api:2.15.0",
```

Comment:
Should we add a comment why these versions are explicitly pinned? Otherwise I fear this will never be removed again, even when we would no longer need it.

Reply:
👍 Maybe add `# log4j deps are added only for https://github.com/bazelbuild/rules_jvm_external/issues/630`

Reply:
Done